dashboards-observability/public/components/visualizations/charts/stats/stats.tsx

```diff
@@ -128,10 +139,7 @@ export const Stats = ({ visualizations, layout, config }: any) => {
           {
             ...STATS_ANNOTATION,
             x: 0 + ANNOTATION_MARGIN_LEFT,
-            y:
-              index > 0
-                ? (index + 1) / metricsLength
-                : 1 / metricsLength,
+            y: index > 0 ? (index + 1) / metricsLength : 1 / metricsLength,
```

Comment:
I saw we have more than one similar calculations here like `(index + 1) / metricsLength : 1 / metricsLength` could we extract all similar logics for this type of calculation to a function for all places that need this or similar calculations?

Reply:
replaced with const as this one is used in this single function only. although revised similar patterns on other code blocks also